src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the `Person` component is correctly defined and exported in the `./components/Person/Person` file. Verify that it can handle cases where the `age` property is missing, as seen with the `olya` object.